src/common/kafka/serializer/BinAggregation.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2024 European Spallation Source, ERIC. see LICENSE file
```

Comment:
This seem to be a generic math abstraction. Maybe it should belong to a 
math library for example <common/math>  ?
And then not using the serializer namespace but essmath namespace or something?